### ISSUE TITLE
ImportError egg-info file folder in Python 2

### ISSUE THREAD

**Issue by ccordoba12** [spyder-ide/spyder-notebook#112]
@lihka1 commented on [Thu Aug 17 2017](https://github.com/spyder-ide/spyder/issues/4983)

## Description

**What steps will reproduce the problem?**

1. 
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "/home/akhil/anaconda2/lib/python2.7/site-packages/spyder/otherplugins.py", line 79, in _import_plugin
    module = _import_module_from_path(module_name, plugin_path)
  File "/home/akhil/anaconda2/lib/python2.7/site-packages/spyder/otherplugins.py", line 101, in _import_module_from_path
    info = imp.find_module(module_name, [plugin_path])
ImportError: No module named spyder_notebook-0.1.1-py2.7.egg-info

## Version and main components

* Spyder Version: 3.2.1 
* Python Version: 2.7.13
* Qt Versions: 5.6.2, PyQt5 5.6 on Linux

## Dependencies
```
pyflakes >=0.5.0  :  1.5.0 (OK)
pycodestyle >=2.3 :  2.3.1 (OK)
pygments >=2.0    :  2.1.3 (OK)
pandas >=0.13.1   :  0.19.2 (OK)
numpy >=1.7       :  1.13.1 (OK)
sphinx >=0.6.6    :  1.5.1 (OK)
rope >=0.9.4      :  0.9.4 (OK)
jedi >=0.9.0      :  0.9.0 (OK)
psutil >=0.3      :  5.0.1 (OK)
nbconvert >=4.0   :  4.2.0 (OK)
sympy >=0.7.3     :  1.0 (OK)
cython >=0.21     :  0.25.2 (OK)
qtconsole >=4.2.0 :  4.2.1 (OK)
IPython >=4.0;<6.0:  5.1.0 (OK)
pylint >=0.25     :  1.6.4 (OK)

```

---

@ccordoba12 commented on [Thu Aug 17 2017](https://github.com/spyder-ide/spyder/issues/4983#issuecomment-323078053)

How did you install Spyder notebook?

---

@lihka1 commented on [Fri Aug 18 2017](https://github.com/spyder-ide/spyder/issues/4983#issuecomment-323272633)

conda install spyder-notebook -c spyder-ide

---

@ccordoba12 commented on [Fri Aug 18 2017](https://github.com/spyder-ide/spyder/issues/4983#issuecomment-323375400)

Ok, this is an error with Spyder notebook, so I'm moving this issue to the corresponding repo.

**Comment by dalthviz** [comment]
Hi @lihka1 could you try to delete the directory `spyder_notebook-0.1.1-py2.7.egg-info` from your env? should be in `/home/akhil/anaconda2/lib/python2.7/site-packages/`

**Comment by dalthviz** [comment]
Note: maybe this is relevant here https://stackoverflow.com/a/24123086

**Comment by ccordoba12** [comment]
@dalthviz, is that file part of our wheels?

**Comment by dalthviz** [comment]
I think the `.egg-info` is in the dist (source). In the wheel there is a `.dist-info`. I think the conda packages take the source of the pypi package. Maybe there is some option in the recipe of the conda package to ignore the `.egg-info`?

**Comment by lihka1** [comment]
Hi @dalthviz, i just removed _spyder_notebook-0.1.1-py2.7.egg-info_ as you said.It worked.Thank you.

**Comment by dalthviz** [comment]
This has been fixed in Spyder 3.2.2 by issue [#5128](https://github.com/spyder-ide/spyder/pull/5128)
